### DISCUSSION TITLE
los schemasnodes de las properties puede existir el campo gps?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#192] [2. Questions and Answers] [answered]
Es una pregunta simple si puede tener los schemasnodes de las properties para colocar el gps.

Gracias :)

**Comment by acedward** [comment, marked as answer]
Los schemaNodes pueden guardar la información que quieras. Como el "GPS" representa una ubicación geográfica descrita por dos números: `latitude` y `longitud`.

Esto solo crea los campos, ahora tú eres el encargado de llenarlos.

**Reply by NicolasPLG** [reply]
hola @aceward, entiendo. entonces debería crear 2 schemas numéricos y a cada uno colocar el latitude y longitud, y la respuesta sería el mismo campo, pero agregaría (por decir) |latitude o |longitud.

**Reply by acedward** [reply]
Solo serían dos campos numericos. Uno llamado latitude y otro longitud. Ambos de tipo numético.

**Reply by NicolasPLG** [reply]
gracias @acedward
